### ISSUE TITLE
Group principles with reasoning

### ISSUE THREAD

**Issue by cstettler** [ISA-Principles/isa-principles.org#9]
Currently, the ISA principles are grouped on one page, but separated from the reasoning page. This may make it difficult for readers (i.e. me) to remember the exact statement about principle 8 when reading the corresponding reasoning 8 ...

As a potential improvement, the reasoning could be placed directly underneath the corresponding principle (first saying "what", then "why"), making the reasoning page in its current form obsolete.

Optionally, one page per principle with corresponding reasoning could also work.

**Comment by ewolff** [comment]
Good point. We could have a heading for each principle and then link from the index to the reasoning and vice versa. I don't think a separate page per principle makes a lot of sense. The site would provide no overview any more.
